src/color_wheel.cpp

```diff
@@ -42,6 +43,29 @@ static const ColorWheel::DisplayFlags hard_default_flags = ColorWheel::SHAPE_TRI
 static ColorWheel::DisplayFlags default_flags = hard_default_flags;
 static const double selector_radius = 6;
 
+struct RingEditor
+{
+    double hue_diff;
+    bool editable;
+    int symmetric_to;
+    int opposite_to;
+    RingEditor(double hue_diff_, bool editable_, int symmetric_to_=-1, int opposite_to_=-1) :
```

Comment:
Perhaps. This is just my style :) (`hue_diff(hue_diff)` looks a bit weird to me)